src/main/java/seedu/module/ui/ModuleListPanel.java

```diff
@@ -39,7 +40,9 @@ protected void updateItem(Module trackedModule, boolean empty) {
                 setGraphic(null);
                 setText(null);
             } else {
-                setGraphic(new ModuleCard((ModuleTracked) trackedModule, getIndex() + 1).getRoot());
+                if (trackedModule instanceof TrackedModule) {
+                    setGraphic(new ModuleCard((Trackable) trackedModule, getIndex() + 1).getRoot());
+                }
```

Comment:
This is still broken. Just a brief overview: the `find` command will display a list of `ArchivedModule`s. The above change breaks the display of `ArchivedModule` because it doesn't implement `Trackable`. Here's a screenshot of the issue:
![image](https://user-images.githubusercontent.com/43642522/66838929-2b2dcc80-ef98-11e9-8c04-25b3b4de6875.png)


I suggest that a modification should be made in the `ModuleCard` class, which should take in a `Module` to display, and subsequently type check for `Trackable` and display contents belonging to `Trackable` (which should be things like deadlines, external links, grades, etc)